examples/GHZsetup/GHZsetup.jl

```diff
@@ -0,0 +1,92 @@
+using ResumableFunctions
+using ConcurrentSim
+using Revise
+using QuantumSavory
+using QuantumSavory.ProtocolZoo
+const bell = StabilizerState("XX ZZ")
+const perfect_pair = (Z1⊗Z1 + Z2⊗Z2) / sqrt(2)
+const perfect_pair_dm = SProjector(perfect_pair)
+const mixed_dm = MixedState(perfect_pair_dm)
+
+noisy_pair_func_depol(p) = p*perfect_pair_dm + (1-p)*mixed_dm
+
+function noisy_pair_func(F)
+    p = (4*F-1)/3
+    return noisy_pair_func_depol(p)
+end
+
+S = 5 # number of sensors
+F = 0.99 # fidelity
+
+noisy_pair = noisy_pair_func(F)
+
+@resumable function GHZ_projection(sim, net, S; time=0.1)
```

Comment:
maybe adding a line here that says `hub_node = net[S]` would make the code a bit simpler